tagger/plot/common.py

```diff
@@ -0,0 +1,62 @@
+#Plotting
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib
+import mplhep as hep
+plt.style.use(hep.style.ROOT)
+import matplotlib.pylab as pylab
+params = {'legend.fontsize': 'medium',
+         'axes.labelsize': 'x-large',
+         'axes.titlesize':'x-large',
+         'xtick.labelsize':'medium',
+         'ytick.labelsize':'medium'}
+pylab.rcParams.update(params)
+
+#GLOBAL VARIABLES TO USE ACROSS PLOTTING TOOLS
+MINBIAS_RATE = 32e+3 #32 kHZ
```

Comment:
Why this rate?

Reply:
Not every bunch is filled at the LHC, this has been the number I used to derive the PuppitauNN rates, which has been pretty much verified by the menu team. Unfortunately I found no official source for this haha, I was told by Phil to use this number. I think we could definitely follow up on this in the future with the appropripate people.